extensions/amp-ad/amp-ad.md

```diff
@@ -89,6 +89,7 @@ resources in AMP. It requires a `type` argument that select what ad network is d
 - [Atomx](../../ads/atomx.md)
 - [Chargeads](../../ads/chargeads.md)
 - [Colombia](../../ads/colombia.md)
+- [Content.ad](../ads/contentad.md)
```

Comment:
@jlucero-contentad seems this is the cause. it missed extra `../`. No worry, I can fix that for you.